config/manifests/bases/camel-k.clusterserviceversion.yaml

```diff
@@ -151,6 +151,7 @@ spec:
   minKubeVersion: 1.11.0
   provider:
     name: The Apache Software Foundation
+  replaces: camel-k-operator.v1.4.0
```

Comment:
It can be removed in this bases csv or indeed in the created bundle. It is problematic because if a 1.4.0 did not exist in the OLM catalog then this CSV would fail to index (tried it with fuse-online). At this moment, this attempts to duplciate the currently generating bundle. If we need something more dynamic then need to discuss further.

Reply:
This is the base CSV, and is only used to generate the local OLM bundle. It serves as the base for downstream versions, like https://github.com/operator-framework/community-operators and product versions, that have their own lineage. And that prevents developers from testing the local OLM bundle: https://github.com/apache/camel-k/issues/1923#issuecomment-763448469.

So based on this, I'd be inclined to leave it unset, and let downstream bundles add it.